packages/integration-tests/src/process-controller/predicated_action_macros.ts

```diff
@@ -45,8 +46,8 @@ export const rejectCleanupSandbox = () =>
  * Reusable predicated action: Wait for sandbox to become idle and then update the
  * backend code which should trigger sandbox again
  */
-export const updateFileContent = (from: URL, to: URL) => {
-  return waitForSandboxToBecomeIdle().updateFileContent(from, to);
+export const updatePathContent = (replacements: PathReplacement[]) => {
+  return waitForSandboxToBecomeIdle().updatePathContent(replacements);
```

Comment:
nit: we are not updating the path but the file. Perhaps `CopyOperation[]`?

Reply:
`CopyOperation` works, will update